generators/Exercises/DifferenceOfSquares.cs

```diff
@@ -0,0 +1,28 @@
+using Generators.Input;
+using System;
+
+namespace Generators.Exercises
+{
+    public class DifferenceOfSquares : Exercise
+    {
+    	protected override void UpdateCanonicalData(CanonicalData canonicalData)
+    	{
+    		foreach	(var canonicalDataCase in canonicalData.Cases)
+    			switch(canonicalDataCase.Property)
+            	{
+            		case "squareOfSum":
+            			canonicalDataCase.Property = "CalculateSquareOfSum";
+            			break;
+            		case "sumOfSquares":
+            			canonicalDataCase.Property = "CalculateSumOfSquares";
+            			break;
+            		case "differenceOfSquares":
+            			canonicalDataCase.Property = "CalculateDifferenceOfSquares";
+            			break;
+                    default:
+                        canonicalDataCase.Property = "Default_This_Shouldnt_Happen";
```

Comment:
Could you throw an exception here?

Reply:
The formatting of this switch is off (see the diff in GitHub).

Reply:
Oh. Um, that `default` shouldn't even exist, actually. I just put it there to debug my changes to the property names. If another property comes, and has a new name, it shouldn't make the generator throw. Since this loop is an override of the original names, it should just ignore new names that come up.

Reply:
I'll change that up